### ISSUE TITLE
can you add save responses to text file ?

### ISSUE THREAD

**Issue by L3GEND-IQ** [Abhilash-Ram/RestClient#1]
hello thank you for this project it's great 

can you add save responses to text file ?

i mean json responses svaed to text file

**Comment by Abhilash-Ram** [comment]
Yes. Thank you for showing interest. it just started with an idea to know about how rest works. will continue with your idea. :-)

**Comment by L3GEND-IQ** [comment]
thank you so much 👍
